Mathlib/Analysis/LocallyConvex/Basic.lean

```diff
@@ -278,6 +282,20 @@ protected theorem Balanced.convexHull (hs : Balanced 𝕜 s) : Balanced 𝕜 (co
   simp only [smul_add, ← smul_comm]
   exact convex_convexHull ℝ s (hx a ha) (hy a ha) hu hv huv
 
+variable (F ℱ : Type*) [SeminormedAddCommGroup F] [Module 𝕜 F]
+variable [FunLike ℱ F E] [LinearMapClass ℱ 𝕜 F E]
+
+omit [Module ℝ E] in
```

Comment:
Rather than omitting this, I think you should add `in` at the end of line 274 instead.